operator/src/main/java/oracle/kubernetes/operator/helpers/JobHelper.java

```diff
@@ -262,6 +269,36 @@ private boolean isIntrospectionNeeded(Packet packet) {
               || isIntrospectVersionChanged(packet);
       }
 
+      private String introspectionNeededReason(Packet packet) {
+        StringBuilder sb = new StringBuilder("introspection needed because ");
+        if (getDomainTopology() == null) {
+          sb.append("domain topology is null");
+        } else if (isBringingUpNewDomain(packet)) {
+          sb.append("bringing up new domain");
+        } else {
+          sb.append("something else");
+        }
+        return sb.toString();
+      }
+
+      private String introspectionNotNeededReason(Packet packet) {
+        StringBuilder sb = new StringBuilder("introspection not needed because ");
```

Comment:
How do we use these strings? I'm fine with using non-internationalizable strings for debug messages, but not for anything used in INFO for greater logs.

Reply:
Ignore... I see where you are using these messages. I wouldn't have asked if ™️ the work "Debug" was in the method name.

Reply:
I built this to debug fiber actions, and currently the only way to enable it is to change the code to add a prefix, such as
```
    @Override
    public NextAction apply(Packet packet) {
      packet.put(Fiber.DEBUG_FIBER, "REG-> (CRD)");
      return doNext(context.verifyCrd(getNext()), packet);
    }
```

which I had added to CrdHelper. There is not any current way for a user to enable it, and it would make little sense to somebody without access to the source code in any event. That said, I have a longer-term idea that might make it usable for certain end-users, and if we went in that direction, internationalization would absolutely make sense.